### PR TITLE
fix breaking change can't find file bug

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export const getTargetBranch = function() {
 export const doOnTargetBranch = async <T>(pr: devOps.PullRequestProperties, func: () => Promise<T>) => {
   const currentDir = process.cwd();
 
-  pr.checkout(pr.targetBranch)
+  await pr.checkout(pr.targetBranch)
```
Comment (jianyexi) [inline src/utils.ts]: could you pls write a test for it ?

Comment (ruowan) [reply src/utils.ts]: Ok. I try to add a test.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.6.7
+
+- Fixed breaking change can't find file bug.
```
Comment (jianyexi) [inline CHANGELOG.md]: could you pls make it more easy to understand ?

Comment (akning-ms) [reply CHANGELOG.md]: yes. e.g. add some description  why cause this issue

Comment (ruowan) [reply CHANGELOG.md]: Updated. Add root cause analysis.

File: src/tests/helper.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  await gitRemote({ config: ["user.email", "test@example.com"] });
+  await gitRemote({ config: ["user.name", "test"] });
+
+  // commit invalid 'specification/readme.md' to 'master'.
+  const specification = path.join(remote, "specification");
+  await pfs.mkdir(specification);
+  await pfs.writeFile(path.join(specification, "readme.md"), "");
+  await pfs.writeFile(
+    path.join(specification, "file1.json"),
+    `
```
Comment (raych1) [inline src/tests/helper.ts]: Can we move all the inline JSON as separate file?

Comment (ruowan) [reply src/tests/helper.ts]: Do you mean after running the test we need to make a cleanup.

Comment (ruowan) [reply src/tests/helper.ts]: This is just for mocking git repo . Only used once. So it's not needed to move inline json to file. 

Added cleanUp function.

File: src/tests/utilsTest.ts [hunk 4]
```diff
@@ -36,7 +37,9 @@ import * as fs from 'fs-extra'
     }
 
     @test async "TestDoOnTargetBranch" () {
-        const cfg = await createDevOpsEnv('devops');
+        const rootName = 'test-root'
```
Comment (raych1) [inline src/tests/utilsTest.ts]: Seems the style is inconsistent, some rows are not ended by `;`, does it have tslint check.

File: src/tests/helper.ts (hunk 3 above)
Comment (raych1) [reply src/tests/helper.ts]: Sounds good.

File: src/tests/utilsTest.ts (hunk 4 above)
Comment (ruowan) [reply src/tests/utilsTest.ts]: The repo doesn't have any tslint check. I will create a separate PR to fix.

**Pull request merged**